build-caching-maven-samples/spotless-project/pom.xml

```diff
@@ -0,0 +1,183 @@
+                                                </properties>
+                                            </inputs>
+                                        </property>
+                                        <!-- Entities used by Spotless with parameters defined in FormatterFactory -->
+                                        <property>
+                                            <name>java</name>
+                                        </property>
+                                        <property>
+                                            <name>python</name>
+                                        </property>
+                                        <property>
+                                            <name>scala</name>
+                                        </property>
+                                        <property>
+                                            <name>groovy</name>
+                                        </property>
+                                        <property>
+                                            <name>javascript</name>
+                                        </property>
+                                        <property>
+                                            <name>json</name>
+                                        </property>
+                                        <property>
+                                            <name>kotlin</name>
+                                        </property>
+                                        <property>
+                                            <name>cpp</name>
+                                        </property>
+                                        <property>
+                                            <name>antlr4</name>
+                                        </property>
+                                        <property>
+                                            <name>sql</name>
+                                        </property>
+                                        <property>
+                                            <name>typescript</name>
+                                        </property>
+                                        <property>
+                                            <name>yaml</name>
+                                        </property>
+                                        <property>
+                                            <name>pom</name>
+                                        </property>
+                                        <property>
+                                            <name>markdown</name>
+                                        </property>
+                                    </nestedProperties>
+                                    <outputs>
+                                        <directories>
+                                            <directory>
+                                                <name>buildDir</name>
```

Comment:
i wonder if there are any other outputs to cache, like reports? or are those all in here?

Reply:
I verified again and I only saw https://github.com/diffplug/spotless/blob/main/plugin-maven/src/main/java/com/diffplug/spotless/maven/AbstractSpotlessMojo.java#L352